modules/navBar.js

```diff
@@ -0,0 +1,29 @@
+// Nav Bar
+const navList = document.querySelector('.navList');
+const navAdd = document.querySelector('.navAdd');
+const navContact = document.querySelector('.navContact');
+const infoPage = document.querySelector('.info-page');
+const addBookPage = document.querySelector('.add-book-page');
+const contact = document.querySelector('.contact');
+
+navList.addEventListener('click', () => {
+  infoPage.classList.remove('close');
+  addBookPage.classList.add('close');
+  contact.classList.add('close');
+});
+
+navAdd.addEventListener('click', () => {
+  infoPage.classList.add('close');
+  addBookPage.classList.remove('close');
+  contact.classList.add('close');
+});
+
+navContact.addEventListener('click', () => {
+  infoPage.classList.add('close');
+  addBookPage.classList.add('close');
+  contact.classList.remove('close');
+});
```

Comment:
- [OPTIONAL] Your app looks good, but the first time the user loads the page and when the user reloads the page, all the sections appear on screen, making it look no professional. It would be great if you fix this behavior, making one of the sections appear the first load, and the others appear only when the user clicks on the nav link.
 - 
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/61919617/214404861-ec1cd9d3-6458-492d-b620-f6379cc9a39f.png">